docs/users/developer-tools.md

```diff
@@ -38,6 +38,8 @@ To execute a fully-featured `composer create-project` command, you can execute t
 
 `ddev exec composer create-project ...`
 
+Note: if you use `composer global` command, the packages will be installed in the user's home directory and will disappear on the next container restart, requiring rerun of the command. You may need an additional step of synchronizing created composer configuration and installed packages with the DDev's [homeadditions folder](extend/in-container-configuration.md) on the host.
```

Comment:
```suggestion
Note: if you run `ddev composer global` command, the global packages will be installed in the in-container user's home directory ( ~/.composer) and will disappear on the next container restart, requiring rerun of the command. You may need an additional step of synchronizing created composer configuration and installed packages with the DDev's [homeadditions folder](extend/in-container-configuration.md) on the host.
```

Reply:
Actually, the composer global happens whether you run it with ddev or inside the container directly. But I am ok with either way.